packages/docs/content/utilities/text/text-color.mdx

```diff
@@ -2,9 +2,9 @@
 title: Text Color
 ---
 
-Use the text color utility to change the default text color of an element.
+import ColorContrastGuidelines from '../../../src/reusable-text-snippets/ColorContrastGuidelines.mdx';
 
-The options below offer color palette combinations that fall within the range of Section 508 compliant and WCAG 2.0 level AA standards which is a minimum color contrast ratio of 4.5:1.
+Use the text color utility to change the default text color of an element.
```

Comment:
removed the a11y paragraph here because its wrong. tried correcting the color pairing examples, but theming makes that difficult (each theme as a different color assigned to each utility 😵 ). 

i can roll back this change and wait until we have a ticket for fixing this page in its entirety, but it seemed like keeping bad content around due to process (waiting for a ticket/prioritization) might result in teams accidentally assuming all these pairings are gtg.

Reply:
Yeah, good call. It's a wish-list item to come up with approved background and foreground combos for each theme